.github/workflows/release.yml

```diff
@@ -38,3 +38,15 @@ jobs:
           ref: main
           workflow: update-eas-build-modules.yml
           token: ${{ secrets.EXPO_BOT_PAT }}
+  open-pr-in-eas-cli:
+    name: Open a PR with version bumps in eas-cli
+    needs: publish-to-npm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Open a PR with version bumps in eas-cli
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          repo: expo/eas-cli
+          ref: main
+          workflow: upgrade-eas-build-modules.yml
```

Comment:
Does this workflow exist in the EAS CLI repo? I mean this action (in this repo) is actually just dispatching the action called `upgrade-eas-build-modules` in the EAS CLI repo

Reply:
I guess your review from 4 minutes later answers this question? haha https://github.com/expo/eas-cli/pull/2216#pullrequestreview-1867416216